<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFlight</class>
 <widget class="QDialog" name="EditFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Flight</string>
  </property>
  <property name="styleSheet">
   <string>background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #87CEEB, stop:1 #FFFFFF);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string>color: #1E3A8A; text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.2);</string>
      </property>
      <property name="text">
       <string>Edit Flight</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>From</string>
        </property>
        <property name="styleSheet">
         <string>color: #1E3A8A;</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fromInput">
        <property name="styleSheet">
         <string>background-color: #FFFFFF; border: 2px solid #4A90E2; border-radius: 8px; padding: 5px; color: #333333;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>To</string>
        </property>
        <property name="styleSheet">
         <string>color: #1E3A8A;</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="toInput">
        <property name="styleSheet">
         <string>background-color: #FFFFFF; border: 2px solid #4A90E2; border-radius: 8px; padding: 5px; color: #333333;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="priceInput">
        <property name="minimum">
         <double>0.0</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="departureDateLabel">
        <property name="text">
         <string>Departure Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="departureDateInput">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="returnDateLabel">
        <property name="text">
         <string>Return Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="returnDateInput">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="airlineLabel">
        <property name="text">
         <string>Airline</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="airlineInput">
        <item>
         <property name="text">
          <string>AirCanada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TurkishAirlines</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="styleSheet">
       <string>QPushButton:enabled[text="OK"] { background-color: #4A90E2; color: white; border-radius: 8px; padding: 8px; min-width: 100px; }
QPushButton:enabled[text="Cancel"] { background-color: #F5A623; color: white; border-radius: 8px; padding: 8px; min-width: 100px; }</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditFlight</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFlight</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>